scripts/create_keystore.sh

```
@@ -0,0 +1,41 @@
+echo "3. Your name and organizational details"
+echo ""
+
+keytool -genkey -v \
+  -keystore "$KEYSTORE_PATH" \
+  -keyalg RSA \
+  -keysize 2048 \
+  -validity 10000 \
+  -alias "$KEY_ALIAS" \
+  -storetype JKS
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)